include/AutomationTrack.h

```diff
@@ -36,9 +36,11 @@ class AutomationTrack : public Track
 {
 	Q_OBJECT
 public:
-	AutomationTrack( TrackContainer* tc, bool _hidden = false );
+	AutomationTrack(TrackContainer* tc, bool hidden = false);
```

Comment:
Haven't looked at the rest of the code but i believe it makes sense to progress on removing global automation before handling this case.

Reply:
This code and the variable `hidden` was always there. I actually didn't even need to do any formatting changes here.